scala/scala.bzl

```diff
@@ -656,3 +656,52 @@ def scala_export_to_java(name, exports, runtime_deps):
     jars = jars,
     runtime_deps = ["@scala//:lib/scala-library.jar"] + runtime_deps
   )
+
+# This auto-generates a test suite based on the passed set of targets
+# we will add a root test_suite with the name of the passed name
+def scala_test_suite(name, srcs = [], deps = [], runtime_deps = [], data = [], resources = [],
+                     scalacopts = [], jvm_flags = [], visibility = None, size = None):
+    ts = []
+    for test_file in srcs:
+        n = "suite_test_%s_%s_test" % (name, test_file)
```

Comment:
I'll add a sanitize, but cases it doesn't care about upper/lower case. And spaces aren't valid seemingly in input file names to scala library at all.